ui/apps/platform/cypress/integration/compliance-enhanced/complianceEnhancedScanConfigs.test.js

```diff
@@ -1,13 +1,54 @@
 import withAuth from '../../helpers/basicAuth';
+import { interactAndWaitForResponses } from '../../helpers/request';
 import { getRegExpForTitleWithBranding } from '../../helpers/title';
 import { getHelperElementByLabel, getInputByLabel } from '../../helpers/formHelpers';
+import { navigateWizardForward } from '../../helpers/wizard';
 
 import {
     visitComplianceEnhancedSchedulesFromLeftNav,
     visitComplianceEnhancedScanConfigs,
     complianceEnhancedScanConfigsPath,
 } from './ComplianceEnhanced.helpers';
 
+function interceptAndMockComplianceIntegrations(callback) {
+    const alias = 'complianceIntegrations';
+    return interactAndWaitForResponses(
+        callback,
+        {
+            [alias]: { method: 'GET', url: '/v2/compliance/integrations' },
+        },
+        {
+            [alias]: { fixture: 'compliance-enhanced/integrations' },
```

Comment:
Previous comment drew my attention to `compliance` versus `compliance-enhanced`

Because no fixtures for classic compliance, what are pro and con

```js
[alias]: { fixture: 'compliance/integrations' },
```
`